lib/reek/report/report.rb

```diff
@@ -139,7 +136,7 @@ def show
     class HTMLReport < Base
       require 'erb'
 
-      def show(target_path = Pathname.new('reek.html'))
+      def show(target_path: Pathname.new('reek.html'))
```

Comment:
My reasoning is: positional arguments in public methods are a strong API restriction, and if they are there they should be the obvious ones (like `from_path(path)`). If something is passed solely (or mostly) for dependency injection / testing then making it a keyword argument means it won’t steal the precious First Positional Argument spot.

(That said, I’m open for reverting this on behalf of ‘this is the obvious argument to this method’; I’m just addressing the broader comment.) :)